Lesson2/src/Homework.java

```diff
@@ -0,0 +1,97 @@
+import java.util.Scanner;
+
+public class Homework {
```

Comment:
Everything looks good to me! Good job

One thing I will ask you to fix - code formatting. Please press ctrl + alt + l after opening this file in intellij, then commit and push changes

Reply:
after this,  please merge this PR